src/bosh-director/spec/unit/db/migrations/director/20161128181900_add_logs_to_tasks_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'db_spec_helper'
+
+module Bosh::Director
+  describe 'adding event_output and result_output to tasks' do
+    let(:db) { DBSpecHelper.db }
+    let(:migration_file) { '20161128181900_add_logs_to_tasks.rb' }
+
+    before { DBSpecHelper.migrate_all_before(migration_file) }
+
+    it 'sets the default value of the event_output and result_output to empty string' do
+      db[:tasks] << {
+        id: 1,
+        state: 'finished',
+        type: 'something',
+        deployment_name: 'test-deployment',
+        timestamp: '2016-04-14 11:53:42',
+        description: 'description',
+      }
+
+      DBSpecHelper.migrate(migration_file)
+
```

Comment:
@barthy1 You can create a default value using the `before_create` [method](https://github.com/jeremyevans/sequel/blob/master/doc/model_hooks.rdoc#basic-usage).

Here's an example of `before_create` in the cloud config model
https://github.com/cloudfoundry/bosh/blob/master/src/bosh-director/lib/bosh/director/models/cloud_config.rb#L5-L7